### PR TITLE
Ci40 example extensions for LoRa with air quality and thermo3 sensors

### PR DESCRIPTION
**Pull request by MarisaAlina**
Adapted application scripts for Ci40 to work with LoRa RF Click Microcontroller and air quality and thermo3 sensor clicks, using TTN LoRaWAN backend to send data to myCayenne mobile & desktop app.

### CODE REVIEW

File: examples/air-example.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Initialise I2C on Mikrobus 1
+i2c.init()
+i2c.select_bus(MIKROBUS_1)
+
+# Read values
+print('{} values from air_quality'.format(air_quality.get_measure(MIKROBUS_1)))
+
+# Release I2C
+i2c.release()
```
Comment (francois-berder) [inline examples/air-example.py:23]: eof please

**Review by francois-berder — COMMENTED** [review]
Hello,

Thanks for your contribution.
I could not test your PR since I do not have a Ci40 running. 
Concerning the air-example, I think this is good and could be merged.
Concerning lora-air4, lora-thermo4 and lora-air examples, I do not really think they fit with the other examples.
The reason why I started writing LetMeCreate/PyLetMeCreate was to make it trivial to write applications on the Ci40. One way to demonstrate that was by having very short examples: max 100 lines, but preferably 20-40 lines.
Besides, you do not use the lora wrapper. Could you change LetMeCreate/PyLetMeCreate lora click to support what you are trying to do ? I believe this would make these examples much shorter.

Also, PR should be done on the dev branch. Could you reopen this PR at https://github.com/Creatordev/pyletmecreate ?

François

**Comment by MarisaAlina** [comment]
Hi Francois,

Thanks for the feedback. I added eof to the air-example so this could be merged.

Regarding your request to change the LetMeCreate/PyLetMeCreate lora click I’m not entirely sure I understood what you mean. 

The lora-air and lora-thermo example are comprehensive scripts that 
a) use the lora wrapper for the Ci40 to communicate with the lora transceiver 
b) establish connection to the TTN LoRaWAN backend and 
c) encode the payload according to the Cayenne LPP format to be displayed in their mobile/desktop app. 

I understand that this exceeds your 40 line limit and is probably using too specific services. I’m happy to rewrite the lora click though, please could you clarify what you were thinking of exactly?

Thanks,
Marisa

**Comment by francois-berder** [comment]
Hello Marisa,

Sorry for the delay.
First thing first, open your PR against this repo: https://github.com/Creatordev/pyletmecreate.
I will merge it there.

Thanks,
François

**Pull request closed without merging**